### PR TITLE
Ga

### PR DESCRIPTION
**Pull request by alicoding**

### CODE REVIEW

**Comment by alicoding** [comment]
@adamlofting for some reason github asks me to create new PR instead of adding another commit and I couldn't figure out why O_o but I have fix everything you've asked me there.

**Comment by alicoding** [comment]
Completes more requisites for #77 .

File: templates/pages/reset-password.jsx [hunk 1]
```diff
@@ -78,16 +81,19 @@ var ResetPassword = React.createClass({
...
       }
       // handle errors!
     }.bind(this)).catch(function(ex) {
+      ga.event({category: 'Reset Password', action: 'Error parsing response from the server'});
       console.error('Error parsing response', ex);
     });
   },
   handleResetPassword: function(error, data) {
     if ( error ) {
+      ga.event({category: 'Reset Password', action: 'Error', label: 'Error parsing response from the server'});
```
Comment (adamlofting) [inline templates/pages/reset-password.jsx]: Is this a server error, or a validation error? (the console.error below reports 'validation')?

File: templates/pages/reset-password.jsx [hunk 2]
```diff
@@ -107,13 +113,20 @@ var ResetPassword = React.createClass({
...
           submitForm: true
         });
+        ga.event({category: 'Reset Password', action: 'Successfully request password reset'});
+
       } else if ( response.status === 400 ) {
+        ga.event({category: 'Reset Password', action: 'Error', label: 'Bad request for request password reset'});
+
         console.error("Bad Request", response.json());
       } else if ( response.status === 401 ) {
+        ga.event({category: 'Reset Password', action: 'Unauthorized for request password reset'});
```
Comment (adamlofting) [inline templates/pages/reset-password.jsx]: Can this be:
```js
ga.event({category: 'Reset Password', action: 'Error', label: 'Unauthorized for request password reset'});
```

File: templates/pages/reset-password.jsx [hunk 3]
```diff
@@ -53,6 +55,7 @@ var ResetPassword = React.createClass({
   },
   handleRequestPassword: function(error, data) {
     if ( error ) {
+      ga.event({category: 'Reset Password', action: 'Error form validation on request password page'});
```
Comment (adamlofting) [inline templates/pages/reset-password.jsx]: Just for consistency with the other pages, can we change this to:
```js
ga.event({category: 'Reset Password', action: 'Error during form validation'});
```

**Comment by adamlofting** [comment]
Thanks @alicoding, this is great work!

There are just two text changes, and one small question from me then this is an R+

File: templates/pages/reset-password.jsx (hunk 1 above)
Comment (alicoding) [inline templates/pages/reset-password.jsx]: This is form validation error

**Pull request merged**